polaris-react/src/components/Modal/components/Dialog/Dialog.scss

```diff
@@ -19,12 +19,19 @@ $dangerous-magic-space-16: 64px;
   display: flex;
   flex-direction: column;
   justify-content: flex-end;
+  // Allow passthrough of click events to dismiss modal on backdrop click
+  pointer-events: none;
 
   @include breakpoint-after($layout-width-page-with-nav-base) {
     justify-content: center;
   }
 }
 
+.Dialog {
+  // Resume pointer events
+  pointer-events: initial;
+}
```

Comment:
I was testing these changes in Storybook and they work 🙌 
but for some reason, in the `Modal with scroll listener`, the click is on the Backdrop is not working
Well it works, only if you click on the margin space at the top and at the bottom, I made a video:

https://user-images.githubusercontent.com/6597467/170225114-9b2e7878-5895-42ed-a004-63dbffb05f64.mp4

---

I checked the CSS and it seems that this code here is the culprit, but I didn't understand why this is only a problem in this specific example. I bet you have more context to find out why

Here is the `pointer-events: none` you added to the `Container` class: 

<img width="714" alt="scrollable-modal-pointer-events-1" src="https://user-images.githubusercontent.com/6597467/170225365-d5fc408b-8a7f-41c6-be9d-a002a6928834.png">


... And here is the child with `pointer-events: initial` that is making the click not work:

<img width="666" alt="scrollable-modal-pointer-events-2" src="https://user-images.githubusercontent.com/6597467/170225385-b140884f-e791-4c80-bef6-f0760e472766.png">

But again this is only a problem in this particular event 🤷 
Do you have an idea of why this is happening?